Source/SHACL/RealEstateCore/rec.ttl

```diff
@@ -16,9 +16,23 @@
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+geojson:MultiPoint
+  rdf:type rdfs:Datatype ;
+  rdfs:label "Point" ;
+  owl:onDatatype xsd:string ;
+.
+geojson:MultiPolygon
+  rdf:type rdfs:Datatype ;
+  rdfs:label "Point" ;
```

Comment:
Should this be: `rdfs:label "MultiPolygon" ;`

Reply:
Yes it should, thanks for noticing! Will update PR.